### ISSUE TITLE
Reproducing UESTC results

### ISSUE THREAD

**Issue by ottj** [PACerv/ImplicitMotion#3]
Thanks for releasing the code of this interesting work!
I tried to reproduce the UESTC results, hence used the config in the UESTC MLP folder and ran it (same parameters, also on A100 cluster  etc., only changed folder to dataset, smpl, and classifier model).
However, I get this as output from the test_metrics script:

`
[2023-09-14 09:27:12] [    INFO] --- Summary Real acc : 0.987 +- 0.000 | fid : 0.146 +- 0.011 | div : 33.337 +- 0.153 | mul : 14.291 +- 0.064 | dis : 4.938 +- 0.008 (test_metrics.py:587)

[2023-09-14 09:27:12] [    INFO] --- Summary acc : 0.885 +- 0.001 | fid : 42.417 +- 0.228 | div : 29.715 +- 0.169 | mul : 13.440 +- 0.054 | dis : 8.065 +- 0.010 (test_metrics.py:589)
`

FID seems to be wrong for the real data, the rest seems to be fine. For the trained model, however the scores esp. accuracy are way of. Do you have an idea what needs to be changed to be closer to the numbers reported in the paper? Do you have a pretrained model that you could share so I can see whether the problem lies with the test script or the training?
